### ISSUE TITLE
Module positions: content_top / content_bottom

### ISSUE THREAD

**Issue by ghost** [GavickPro/Portfolio-Free-Joomla-Template#33]
Hello,

I've looked everywhere to see if anyone had that issue, but i couldn't find an answer.

It's impossible to publish content in the content_top or the content_bottom module positions, even when content is published in the component module.

![module-positions](https://cloud.githubusercontent.com/assets/18574736/14802007/3f7c8032-0b4e-11e6-93ba-5fd975b67d5e.png)

There is something here i just can't get my hand on.

Thanks for your help.

**Comment by dziudek** [comment]
Did you tried to use these module positions on the subpages other than portfolio view? Please see that there is a function which checks it the grid is not displayed before displaying these positions: https://github.com/GavickPro/Portfolio-Free-Joomla-Template/blob/master/index.php#L78

**Comment by ghost** [comment]
You're right: the module is displayed correctly on subpages as long as the component isn't published.

Does it mean that modules can't be published in content_top or content_bottom positions on pages displaying the component module or on the home page?

**Comment by dziudek** [comment]
You cannot display these modules when the portfolio grid is dosplayed.

**Comment by ghost** [comment]
It's good to know. Thank you very much for your answer.
